docs/shared/setup-codegen/single-panel.mdx

```diff
@@ -14,37 +14,37 @@ import {CodeBlock, CodeBlockProps} from "@apollo/chakra-helpers"
 
 #### Initialize the code generation configuration
 
-The Codegen CLI uses a JSON file to configure the code generation engine. Use the Codegen CLI `init` command to create this file with default values.
+The Codegen CLI uses a JSON file to configure the code generation engine. You can use the Codegen CLI's `init` command to create this file with default values.
 
-From your project's root directory, run:
+From your project's root directory, run the following command with your customized values:
 
 <CodeBlock
   code={props.cliPath + " init --schema-name ${MySchemaName} --module-type ${ModuleType}"}
   language="bash"
   showLineNumbers='true'
 />
 
-- `${MySchemaName}` should be the name you want for the namespace of your generated schema files.
-- `${ModuleType}` is used to configure how your generated schema types will be included in your project.
-  > This is an important decision that must be made prior to configuring code generation. To determine the right option for your project, check our our [project configuration documentation](/project-configuration).
+- `${MySchemaName}` provides a name for the namespace of your generated schema files.
+- `${ModuleType}` configures how your generated schema types are included in your project.
+  > This is a crucial decision to make **before configuring code generation**. To determine the right option for your project, see [Project Configuration](../project-configuration).
```

Comment:
Is this relative path going to work correctly from all places? This is a shared mdx component, and it's used from pages at different paths. Not sure if the link gets resolved relative to the path of the shared component, or the path of the page it is included in? @trevorblades would probably know.

If relative won't work here, the correct solution I think is `[Project Configuration](ios/project-configuration)`.

But that isn't future proof once this isn't the latest docs version anymore haha.

Reply:
I believe it is relative to the page that includes it. @trevorblades can you confirm that for us please?